### ISSUE TITLE
[DOC]: How to handle the distributed dataloader when take use of Gemini Zero3 to acclerate?

### ISSUE THREAD

**Issue by mexiQQ** [hpcaitech/ColossalAI#2861]
### 📚 The doc issue

Thanks for your effort on this library. And maybe your product is fantastic.

However, if the examples and documents are not clear and updated on time, it would be disappointing.

And please also explain why Gemini is not compatible with Gradient Accumulation. (While Deepspeed doesn't have this problem)

Thanks.

**Comment by 1SAA** [comment]
Currently, generated gradients will reuse the footprint allocated for parameters to save your memory usage. Thus, it can not run twice since the parameters have changed. You can save more memory compared with DeepseedZero3.

**Comment by binmakeswell** [comment]
We have updated a lot. This issue was closed due to inactivity. Thanks.
